ci-operator/config/openshift/dedicated-admin-operator/openshift-dedicated-admin-operator-master.yaml

```diff
@@ -0,0 +1,24 @@
+base_images:
+  base:
+    name: '4.0'
+    namespace: ocp
+    tag: base
+binary_build_commands: make test build
```

Comment:
`make test` seems like something you'd want in a test target?

Reply:
It's just some simple unit testing, we might split into a separate step if necessary